lib/aws.js

```diff
@@ -1,10 +0,0 @@
-'use strict'
```

Comment:
This file seemed only used for setting the region in central config, but it's not possible AFAIK with the separated clients, so they just set the region individually now.